Hash_map/include/CGAL/Tools/chained_map.h

```diff
@@ -233,8 +239,8 @@ T& chained_map<T, Allocator>::access(chained_map_item p, std::size_t x)
 
 
 template <typename T, typename Allocator>
-chained_map<T, Allocator>::chained_map(std::size_t n)
-  : table(nullptr), reserved_size(n)
+chained_map<T, Allocator>::chained_map(std::size_t n, const T& d)
+  : table(nullptr), reserved_size(n), def(d)
```

Comment:
I think warnings get emitted when the initialisation order here doesn't match the member declaration order.